www/netj00/Semestralni Prace/data/css/layout.css

```diff
@@ -0,0 +1,128 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    margin: 0;
+    padding-top: 0;
+    background-color: #d6d6d6;
+}
+
+header {
+    padding: 30px;
+    font-size: 35px;
+    text-align: center;
+}
+
+section {
+    display: flex;
+}
+
+.nav ul {
+    display: flex;
+    flex-direction: column;
+    width: 180px;
+    background: #ccc;
+    height: 100%;
+    padding: 0;
+    margin: 0;
+}
+
+.nav ul {
+    padding: 0;
+    margin: 0;
+}
+
+.nav li {
+    overflow: auto;
+    padding: 0;
+    margin: 0;
+    list-style-type: none;
+}
+
+.nav li:first-child {
+    position: center;
+}
+
+.logo {
+    display: block;
+    margin-left: 15%;
+    margin-right: auto;
+    margin-top: 1em;
+    width: 100px;
+}
+
+.nav li:nth-child(2) {
+    margin-top: 15px;
+}
+
+.nav li:last-child {
+    margin-top: auto;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.nav li a {
+    color: black;
+    padding: 4px 16px;
+    display: block;
+    text-decoration: none;
+}
+
+.nav li a.active {
+    background-color: #4b4b4b;
+    color: #dbdbdb;
+}
+
+.nav li a:hover:not(.active) {
+    background-color: #555;
+    color: white;
+}
+
+.nav-separator {
+    background: url("../images/separator.png") no-repeat center;
+    height: 1em;
+    max-width: 85%;
+}
+
+.content {
+    margin-left: auto;
+    margin-right: auto;
+    padding: 0 15px 0;
+    max-width: 900px;
+    height: 100%;
+    float: left;
+}
+
+@media print {
+    .nav, img {
+        display: none;
+        height: 0;
```

Comment:
kdyz to ma display none tak proc jeste height:  0

Reply:
Netuším proč ale když img nemá height 0 tak se zobrazuje v printu

Reply:
no protoze u img mas specifictejsi selektor, v print mas obecnejsi selektor a proto to pravidlo v media print ma mensi prioritu
napr. `main article img` a `img`, co je specifictejsi?

Reply:
Hotovo